packages/ilib-lint/src/Project.js

```diff
@@ -63,10 +63,11 @@ const unknownFileTypeDefinition = {
  * @private
  * @param {Object} instance the instance to check
  * @param {String} methodName the name of the method to check
+ * @param {Class} parentClass the parent class of the instance or one of its ancestors
  * @returns {boolean} true if the method is defined in the class itself
  */
-function isOwnMethod(instance, methodName) {
-    return typeof(instance[methodName]) === 'function' && instance[methodName].prototype === instance;
+function isOwnMethod(instance, methodName, formatter) {
```

Comment:
Change the code to go with it...

Reply:
However, in the case of the Formatter class, it is not possible to create an instance.
 ```Error: Cannot instantiate abstract class Formatter!```

Reply:
I updated the code a bit. Please check if this is more appropriate.

Reply:
No, it should not be instantiable

Reply:
What I meant about changing the code is that the jsdoc says the third parameter is called "parentClass", but in the code, it says "formatter". Change that to "parentClass"

Reply:
oooooops, sorry for the misunderstanding.;(  I've updated it.